deploy/04-operator.yaml

```diff
@@ -36,3 +36,5 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "cluster-image-registry-operator"
+            - name: IMAGE
+              value: docker.io/openshift/origin-docker-registry
```

Comment:
This needs to be a exact match to the value in `name` from here https://github.com/openshift/cluster-image-registry-operator/pull/41/files#diff-2a129732795c02bf1b4c8ea7f7645ef1R12

Reply:
thanks, will fix.